<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="qualityCheckBox">
     <property name="text">
      <string>change quality</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="qualitySlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>select image format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="qualitySpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="metadataCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>copy metadata</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="selectFolderPushButton">
     <property name="text">
      <string>select folder</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="folderLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="formatComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>qualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qualityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualitySpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
